index.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const dbhelpers = require("./dbhelpers/helpers");
+
+
+const server = express();
+
+server.get("/api/:whichtable", async (req, res) => {
```

Comment:
As mentioned above, cool use of `whichtable` here and `which` on the other end. Also like seeing `async` / `await` 😄